### PR TITLE
Admin dashboard filter for empty projects

### PR DESCRIPTION
**Pull request by psh0078**
Implement filter for empty projects on admin dashboard.
Empty projects are defined as projects with no commits, or their lastCommit set to null. 

- Need an icon for the filter
- Need standardization to translation

### CODE REVIEW

File: frontend/src/lib/i18n/locales/en.json [hunk 1]
```diff
@@ -207,6 +207,7 @@ the [Linguistics Institute at Payap University](https://li.payap.ac.th/) in Chia
       "title": "Project filters",
       "show_deleted": "Show deleted",
       "hide_drafts": "Hide drafts",
+      "show_empty": "Show empty projects",
```
Comment (rmunn) [inline frontend/src/lib/i18n/locales/en.json:210]: We'd probably prefer "Show only empty projects" here. "Show empty projects" implies that empty projects are hidden by default and this filter option turns them visible, whereas what the filter actually does is hide all projects that are NOT empty. "Hide non-empty projects" would be another possibility, but that's a double negative, which tends to be confusing, so "Show only empty projects" is better language.

File: frontend/src/lib/components/Projects/ProjectFilter.svelte [hunk 2]
```diff
@@ -169,5 +176,16 @@
         </label>
       </div>
     {/if}
+    {#if filterEnabled('emptyProjects')}
+      <div class="form-control">
+        <label class="cursor-pointer label gap-4">
+          <span class="label-text inline-flex items-center gap-2">
+            {$t('project.filter.show_empty')}
+            <!-- Icon -->
```
Comment (rmunn) [inline frontend/src/lib/components/Projects/ProjectFilter.svelte:184]: A good icon to use would be i-mdi-file-hidden, with no color specified.

File: frontend/src/lib/components/Projects/ProjectFilter.svelte [hunk 3]
```diff
@@ -98,6 +100,11 @@
           <Icon icon="i-mdi-script" color="text-warning" />
           {$t('project.filter.hide_drafts')}
         </ActiveFilter>
+      {:else if filter.key === 'emptyProjects' && filter.value}
+        <ActiveFilter {filter}>
+          <Icon icon="i-mdi-script" color="text-warning" />
```
Comment (rmunn) [inline frontend/src/lib/components/Projects/ProjectFilter.svelte:105]: This should be changed to the same icon and text color that you end up using on line 184.

**Review by rmunn — CHANGES_REQUESTED** [review]
Looks good, just need a couple changes to icon and text.

File: frontend/src/lib/i18n/locales/en.json (hunk 1 above)
Comment (psh0078) [reply frontend/src/lib/i18n/locales/en.json]: @myieye Could we use your thought on this please? :)

Comment (myieye) [reply frontend/src/lib/i18n/locales/en.json]: "Show only empty projects" is a decent candidate.
I think my first pick would be "Empty only".

File: frontend/src/lib/components/Projects/ProjectFilter.svelte (hunk 2 above)
Comment (myieye) [reply frontend/src/lib/components/Projects/ProjectFilter.svelte]: I like the icon suggestion 🚀 
I'd be inclined to even go for the i-mdi-folder-hidden.

Comment (myieye) [reply frontend/src/lib/components/Projects/ProjectFilter.svelte]: No worries!

**Pull request merged**